controller_interface/include/controller_interface/chainable_controller_interface.hpp

```diff
@@ -15,6 +15,8 @@
 #ifndef CONTROLLER_INTERFACE__CHAINABLE_CONTROLLER_INTERFACE_HPP_
 #define CONTROLLER_INTERFACE__CHAINABLE_CONTROLLER_INTERFACE_HPP_
 
+#include <fmt/compile.h>
```

Comment:
@mini-1235 As we have included fmt as a build depend only, we will have to place this include in the .cpp, if not we will have to add the dependency to `build_export_depend`. 

For this reason, if there is no changes in the .hpp, please avoid adding the header include there and move it to .cpp